src/Bridge/Symfony/Resources/config/twig.xml

```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
-    <services>
-        <service id="sonata.twig.extension.wrapping" class="Sonata\Twig\Extension\FormTypeExtension">
-            <tag name="twig.extension"/>
-            <argument>%sonata.twig.form_type%</argument>
-        </service>
-        <service id="sonata.twig.status_runtime" class="Sonata\Twig\Extension\StatusRuntime">
-            <tag name="twig.runtime"/>
-        </service>
-        <service id="sonata.twig.status_extension" class="Sonata\Twig\Extension\StatusExtension">
-            <tag name="twig.extension"/>
-        </service>
-        <!-- NEXT_MAJOR: Remove this definition. -->
-        <service id="sonata.twig.deprecated_template_extension" class="Sonata\Twig\Extension\DeprecatedTemplateExtension">
-            <deprecated>The "%service_id%" service is deprecated since sonata-project/twig-extensions 1.4.</deprecated>
```

Comment:
you removed this line, is that normal?

Reply:
Yes, see https://github.com/sonata-project/twig-extensions/pull/124#issue-482833284, maybe I should've split the PR into two. I've removed this service in the second commit https://github.com/sonata-project/twig-extensions/pull/124/commits/5af6ad552d4e3ae4cd7a03fcaaf9b8810d2af199.